vendor/k8s.io/kubernetes/pkg/kubelet/api/v1alpha1/stats/types.go

```diff
@@ -168,7 +168,7 @@ type MemoryStats struct {
 	// dirty memory, and kernel memory. WorkingSetBytes is <= UsageBytes
 	// +optional
 	WorkingSetBytes *uint64 `json:"workingSetBytes,omitempty"`
-	// The amount of anonymous and swap cache memory (includes transparent
+	// The amount of anonymous memory (includes transparent
```

Comment:
> So we need to fix this vendor file, right ?

I guess you just need to checkout the file changes. Seems that you change file manually.

> How / Where do I update the godeps ? sorry I'm new to this.

You should not and need not do this. Just leave the godeps unchanged. :)